src/controllers/AuthController.ts

```diff
@@ -27,21 +26,17 @@ const tokenCookieOpts: express.CookieOptions = {
 }
 
 @singleton()
-@injectable()
```

Comment:
I'm surprised this works removing the `injectable` annotation. I wonder if having `singleton` automatically makes it injectable 🤔 . Why did you remove it?

Reply:
I might be wrong, but I thought that @inject and @injectable being used together, so since we do not inject - removed or de we use it for `env.get`?

Reply:
brought it back, but would be nice to know if it;s related to the `Env` or not

Reply:
So `injectable` is just saying this classes constructor parameters should be injected. This is still the case for `Env` etc.